client/ionic/src/content/endpoints.ts

```diff
@@ -0,0 +1,9 @@
+export const FLOWS_ENDPOINT =
+  'https://worldhealthorganization.github.io/app/content/flows/v1';
```

Comment:
How will this repo be deployed to github pages again? Which branch / folder?

Reply:
Automatically pushed from master for example right now at:  https://worldhealthorganization.github.io/app/identity/who-logo-rgb.png